### PR TITLE
Add home page template

### LINKED ISSUE #12: [Frontend] As a user, I can see dashboard home page
## Why

The user should be able to see the index page, for now with filler text.

## Acceptance Criteria

- The bootstrap template (both CSS and HTML) was set up for the project
- Home page displaying filler text (for now, because we don't have other features yet).

### PR DESCRIPTION
**Pull request by Nihisil**
I'm sorry for the additional notifications, I accidentally pushed to previous PR commits that didn't belong to it. So, I had to recreate PR with only needed commits.

Resolves #12

## What happened

The bootstrap template was set up for the project, with simple home page welcoming text.
 
 \+ I have noticed that GitHub templates (for PRs and issues) were outdated, so I updated them to the latest version from https://github.com/nimblehq/git-template
 
## Proof Of Work

image

### CODE REVIEW

**Review by bterone — APPROVED** [review]

File: assets/css/base/_index.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+body {
+  padding-top: 40px;
```
Comment (bterone) [inline assets/css/base/_index.scss:2]: Small comment, but we tend to use the `rem` function from our `_sizing.scss`. 

```suggestion
  padding-top: rem(40px);
```

File: lib/google_scraping_web/templates/layout/_header.html.heex [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<header class="mb-5">
+  <nav class="navbar navbar-expand-md navbar-dark fixed-top bg-dark">
+    <div class="container-fluid">
+    <a class="navbar-brand" href="/">Google Scraping</a>
+    </div>
```
Comment (bterone) [inline lib/google_scraping_web/templates/layout/_header.html.heex:5]: Small indentation comment

```suggestion
    <div class="container-fluid">
      <a class="navbar-brand" href="/">Google Scraping</a>
    </div>
```

**Review by bterone — APPROVED** [review]
Really minor comments 👍

File: lib/google_scraping_web/templates/layout/_header.html.heex (hunk 2 above)
Comment (Nihisil) [reply lib/google_scraping_web/templates/layout/_header.html.heex:5]: @bterone thanks! Is there any formatter for `.heex` files? I tried to find something, but no luck. What you are using for it?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
-Add the story URL here. Prefer the short link format, e.g. https://app.clubhouse.io/acme/story/1234/
+https://github.com/nimblehq/git-template/issues/??
```
Comment (junan) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: Should be this repo link
```suggestion
https://github.com/Nihisil/nimble-certification-elixir/issues/??
```

**Review by junan — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (Nihisil) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: I'm sorry, for some reason I missed this suggestion. It will be fixed in future PRs 🙏

**Pull request merged**